packages/create-starter/src/utils.ts

```diff
@@ -15,12 +15,15 @@
   }
 }
 
-export async function initGitRepo(path: string) {
+export async function initGitRepo(path: string, packageManager: string) {
   return new Promise((resolve, reject) => {
-    exec(`git init ${path}`, (err) => {
+    exec(`cd ${path} && git init && ${packageManager} install && git add . && git commit -m 'init commit generated by starter.dev CLI' && cd ../`, (err, stdout) => {
```

Comment:
I am having trouble on my end seeing the commit message
<img width="1218" alt="testing-angular-screenshot" src="https://github.com/thisdot/starter.dev/assets/67210629/845cde63-2ca5-4dff-b3e3-09c27c2ae220">

Reply:
I am looking closer at the `package.json` for the angular starter and it looks like there is no yarn dev command. That one has a yarn start command instead. 

That might pose some issues because some of the kits use `yarn dev` while others have `yarn start`. 

We could probably just shorten it to say `yarn` or `npm i`, etc to cover our bases